### DISCUSSION TITLE
plots quality

### DISCUSSION THREAD

**Discussion by mahdipsycho** [Chia-Network/chia-blockchain#6089] [Q&A]
hey every one I had a question
I lately found out that there is a tool for chia using power shell to check plots quality 
some of plots are 30/30 some of them are more than 30/30 and some of them is less than 30/30 
and the main site said don't worry you don't need to remove any plots but the plots are under 0.7 
yeah thats true
but is that mean tha the plot that has score of 0.8 or 0.9 
is the exact same as the plot that has 1.0 score?
so removing 0.8 score plots is just wasting time?
or there is a little different between them?
can some one tell me?
even if there is a little different between 29/30 and 30/30
I wanna know
thanks for every one for reading
and thanks to all of answers

**Comment by tiny0little** [comment]
i had same doubts some time ago. obviously plot with more proof is better. but when you farm via pool, size of your space is considered, not number of proofs you have. quality of plots much more important for solo farming

**Reply by mahdipsycho** [reply]
thanks a lot
You said pool 
is there a good pool for chia?
I heard hpool and uupool are not good ones
is there any other that would work good?

**Reply by tiny0little** [reply]
hpool is only one for now - [miningpoolstats.stream](https://miningpoolstats.stream/chia)

**Reply by padik11** [reply]
> miałem takie same wątpliwości jakiś czas temu. oczywiście spisek z większą ilością dowodów jest lepszy. ale kiedy gospodarujesz przez pulę, pod uwagę brana jest wielkość twojej przestrzeni, a nie liczba posiadanych dowodów. jakość działek znacznie ważniejsza dla rolnictwa indywidualnego

Yes, but if everyone does it, the pool won't win anything, or it will have little chance of winning. Everyone should have the highest quality plot.
